client/src/components/NavBar/index.tsx

```diff
@@ -5,7 +5,9 @@ import FloatingMenu from "../FloatingMenu";
 import "./NavBar.scoped.scss";
 
 const NavBar: React.FC = () => {
-	const { jwt, activeStudyField, signout } = useAuth();
+	const { jwt, activeStudyField, activeOrganization, signout } = useAuth();
+	const baseUrl = `/${activeOrganization?.id}/${activeStudyField?.id}/`;
```

Comment:
I am thinking the /${activeOrganization?.id}/${activeStudyField?.id}/ should move to a function in the auth hook, since it is used widely in depedents of the hook. What do you think?